src/NServiceBus.AcceptanceTests/Sagas/When_using_a_received_message_for_timeout.cs

```diff
@@ -10,15 +10,17 @@
     public class When_using_a_received_message_for_timeout : NServiceBusAcceptanceTest
     {
         [Test]
-        public Task Timeout_should_be_received_after_expiration()
+        public async Task Timeout_should_be_received_after_expiration()
         {
-            return Scenario.Define<Context>(c => { c.Id = Guid.NewGuid(); })
+            var context = await Scenario.Define<Context>(c => { c.Id = Guid.NewGuid(); })
                 .WithEndpoint<ReceiveMessageForTimeoutEndpoint>(g => g.When(session => session.SendLocal(new StartSagaMessage
                 {
                     SomeId = Guid.NewGuid()
                 })))
                 .Done(c => c.TimeoutReceived)
                 .Run();
+
+            Assert.True(context.TimeoutReceived);
```

Comment:
Should we track number of the regular handler invocations to ensure the handler is only called once?